CHANGELOG.md

```diff
@@ -65,6 +65,8 @@ Unreleased changes template.
 * (rules) The version-aware rules have been folded into the base rules and
   the version-aware rules are now simply aliases for the base rules. The
   `python_version` attribute is still used to specify the Python version.
+* (pypi) Updated versions of packages: `pip` to 24.3.1 and
```

Comment:
@aignas I drafted this change in the Changelog. Please confirm if this is OK to you or suggest how to improve it. Thanks!